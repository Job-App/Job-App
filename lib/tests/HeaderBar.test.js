var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");var _react=_interopRequireDefault(require("react"));var _enzyme=require("enzyme");var _Headerbar=_interopRequireDefault(require("../components/HeaderBar/Headerbar.js"));var _this=this,_jsxFileName="C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\tests\\HeaderBar.test.js";test('Testing HeaderBar component',function(){var wrapper=(0,_enzyme.shallow)(_react.default.createElement(_Headerbar.default,{__self:_this,__source:{fileName:_jsxFileName,lineNumber:6,columnNumber:26}}));expect(wrapper).toMatchSnapshot();});