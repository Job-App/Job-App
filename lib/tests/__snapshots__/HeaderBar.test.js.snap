// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HeaderBar component 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#3D348B",
      "flexDirection": "row",
      "height": 111.16666666666667,
      "width": 750,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
          "marginBottom": 10,
          "marginTop": "auto",
        },
        Object {
          "alignItems": "flex-start",
        },
      ]
    }
  >
    <ForwardRef
      activeOpacity={0.8}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginLeft": 24,
          "marginRight": 24,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#E0E2DB",
            "fontSize": 24,
          }
        }
      >
        Add
      </Text>
    </ForwardRef>
  </View>
  <View
    style={
      Array [
        Object {
          "flex": 1,
          "marginBottom": 10,
          "marginTop": "auto",
        },
        Object {
          "alignItems": "flex-end",
        },
      ]
    }
  >
    <ForwardRef
      activeOpacity={0.8}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginLeft": 24,
          "marginRight": 24,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#E0E2DB",
            "fontSize": 24,
          }
        }
      >
        Login
      </Text>
    </ForwardRef>
  </View>
</View>
`;
